### ISSUE TITLE
result on cub_200_2011

### ISSUE THREAD

**Issue by HelloWangg** [mobulan/IELT#6]
HI,I reproduce this work,but I got the accuracy  only 86%,where is the problem?Thanks a lot!

**Comment by LJWERIC1121** [comment]
me too, in cub with 86% accuracy.

**Comment by mobulan** [comment]
> me too, in cub with 86% accuracy.

This was caused by wrong learning rate, the old learning rate was suitable for APEX amp strategy. I have updated the learning rate to fit pytorch amp.
